class.jetpack-idc.php

```diff
@@ -717,4 +716,4 @@ function get_non_admin_contact_admin_text() {
 	}
 }
 
-Jetpack_IDC::init();
+add_filter( 'plugins_loaded', array( 'Jetpack_IDC', 'init' ) );
```

Comment:
I am not sure this should live here.

Reply:
What is your concern?

Reply:
If we ever want to use the autoloader for this file. 
We should extract this part somewhere else. Since loading a class shouldn't have side effects such as adding an action.

Reply:
```suggestion
add_action( 'plugins_loaded', array( 'Jetpack_IDC', 'init' ) );
```